doc/confidence.rst

```diff
@@ -88,6 +88,23 @@ around the best fit value.  For this problem, it is not necessary to
 calculate confidence intervals, and the estimates of the uncertainties from
 the covariance matrix are sufficient.
 
+Working without standard error estimates.
+--------------------------------
+
+Sometimes the estiamtion of the standard errors from the covariance 
+matrix fails, especially if values are near given bounds. Hence, to
+find the confidence intervals in these cases, it is necessary to set 
+the errors by hand. Note that the standard error is only used to find an
+upper limit for each value, hence the exact value is not important.
```

Comment:
I suppose the value for ```stderr``` is kind of important; it looks like the upper/lower values are calculated as -/+ 5*stderr, correct? So the values should be kind of reasonable, and that's why you suggest about 10%?

Reply:
Disclaimer: I have not reread the code I wrote years. IRC, they are just used as the step-size to find an upper limit where Prob(val) > max(sigma). If the step is not large enough, it will double the interval. Hence to work, it is only necessary that P(val) is sensitive to the change.

Reply:
neither am I.... I am fine with this phrase, please just correct the typos I pointed out ;)